packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -41,26 +41,26 @@
          - showing the doorhanger. -->
       <input class="hidden" required />
 
-      <div class="button-row">
-        <button id="submit-btn" {{^isPasswordNeeded}}class="use-logged-in"{{/isPasswordNeeded}} type="submit">{{#t}}Sign in{{/t}}</button>
+      <div class="flex">
+        <button id="{{^isPasswordNeeded}}use-logged-in"{{/isPasswordNeeded}}{{#isPasswordNeeded}}submit-btn{{/isPasswordNeeded}}" class="cta-primary cta-xl {{^isPasswordNeeded}}use-logged-in{{/isPasswordNeeded}}" type="submit">{{#t}}Sign in{{/t}}</button>
```

Comment:
Can you confirm the stray `"` in the `{{^isPasswordNeeded}}use-logged-in"{{/isPasswordNeeded}}` block?

Reply:
Laser eyes! Thanks, that shouldn't have been in there.